packs/data/equipment.db/feng-huo-lun.json

```diff
@@ -0,0 +1,106 @@
+{
+    "_id": "yuki2nCn7e3KStMn",
+    "img": "systems/pf2e/icons/default-icons/weapon.svg",
+    "name": "Feng Huo Lun",
```

Comment:
Remove. Just a rename for the `Wind and Fire Wheel`

Reply:
Removed it, we can migrate later.